documentation/collectionrules/TriggerShortcuts.md

```diff
@@ -0,0 +1,201 @@
+
+### `CPUUsage` Trigger Shortcut
+
+Execute a trigger when the target application's CPU Usage is continuously greater than or less than a specified value for a set duration of time.
+
+#### Properties
+
+| Name | Type | Required | Description | Default Value | Min Value | Max Value |
+|---|---|---|---|---|---|---|
+| `GreaterThan` | double? | false | The threshold level as a percentage that the counter must maintain (or higher) for the specified duration. | `50` | `0` | `100` |
+| `LessThan` | double? | false | The threshold level as a percentage that the counter must maintain (or lower) for the specified duration. If `LessThan` is specified, the default value of `GreaterThan` becomes `null`. | `null` | `0` | `100` |
+| `SlidingWindowDuration` | false | TimeSpan? | The sliding time window in which the counter must maintain its value as specified by the threshold levels in `GreaterThan` and/or `LessThan`. | `"00:01:00"` (one minute) | `"00:00:01"` (one second) | `"1.00:00:00"` (1 day) |
+
+#### Equivalent EventCounter Trigger
+
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "Trigger": {
+      "Type": "EventCounter",
+      "Settings": {
+        "ProviderName": "System.Runtime",
+        "CounterName": "cpu-usage",
+        "GreaterThan": 50
+      }
+    }
+  }
+  ```
+</details>
+
+<details>
+  <summary>Environment Variables</summary>
+  
+  ```bash
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Type="EventCounter"
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Settings__ProviderName="System.Runtime"
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Settings__CounterName="cpu-usage"
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Settings__GreaterThan="50"
+  ```
+</details>
+
+#### Example
+
+Usage that is satisfied when the CPU usage of the application is higher than 50% for a 1 minute window.
+
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "Trigger": {
+      "Type": "CPUUsage"
+    }
+  }
+  ```
+</details>
+
+<details>
+  <summary>Environment Variables</summary>
+  
+  ```bash
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Type="CPUUsage"
+  ```
+</details>
+
+### `GCHeapSize` Trigger Shortcut
+
+Execute a trigger when the target application's GC Heap Size is continuously greater than or less than a specified value for a set duration of time.
+
+#### Properties
+
+| Name | Type | Required | Description | Default Value | Min Value | Max Value |
+|---|---|---|---|---|---|---|
+| `GreaterThan` | double? | false | The threshold level in MBs that the counter must maintain (or higher) for the specified duration. | `10` | `0` | |
```

Comment:
Did we want a larger default here? I understand this is not prescriptive guidance, but 10 feels like something we will simply always trigger on. I would expect a default asp.net core application to **not** trigger the default.

Reply:
I'm open to setting these defaults to anything since I don't have a good intuition for what they should be, but I actually look at it the opposite way - if I'm using a shortcut for the first time and the default is too aggressive, I might be wondering why it didn't work (or if it did work). If it is a pretty easy mark to hit, then I'll know it worked, in which case my next question will be - how can I make this trigger _less_, in which case they can add their own `GreaterThan` value.